process-engine/process_model.service
# ✨Add super admin claim

## Changes

1. Add a claim check for the `can_manage_process_instances` claim, which indicates a super admin.
    - These admins can always see everything, regardless of their LaneClaims.
2. Replace TSLint with ESLint
3. Replace Gulp with a purely TSC based build
4. Apply new ESLint styleguide
5. Remove unhelpful code docs
6. Allow linter errors to cause Jenkins build failures

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/336

PR: #3

## How to test the changes

Best use the [Runtime](https://github.com/process-engine/process_engine_runtime/pull/338) and our [IdentityServer](https://github.com/process-engine/identity_server)

- Try reading a ProcessModel as User `alice` (has the super admin claim)
- Try reading a ProcessModel as User `bob` (is a normal user)
- Alice should always be able to request all ProcessModels
- Bob should only be able to read those ProcessModels where he has sufficient LaneClaims
